test/gpu/octree/test_radius_search.cpp

```diff
@@ -57,6 +57,64 @@
 
 using namespace pcl::gpu;
 
+void
+verifyResults(std::vector<std::vector<int>>& host_search,
+                std::vector<std::vector<float>>& host_sqr_distances,
```

Comment:
Shouldn't these be const as well?

Reply:
I cant make these const because they're assigned to `result_host` and `sqr_dist_host` here, and later sorted.
```cpp
        std::vector<int>& results_host = host_search[i];
        std::vector<float>& sqr_dist_host = host_sqr_distances[i];
```